roles/kong_dns/tasks/print_kong_records.yaml

```diff
@@ -27,7 +27,7 @@
     kong_dns_records: "{{ kong_dns_records + '*.' + item.ingress_class + '.' + cluster_fqdn + '\t\t\t IN\t A\t ' + kong_ip + '\n' }}"
   vars:
     kong_dns_records: ""
-    instance_kong_proxy: "kong-tavros-{{ item.name }}-kong-proxy"
+    instance_kong_proxy: "kong-tavros-{{ item.name }}{{'-dp' if item.hybrid }}-kong-proxy"
```

Comment:
is this something that's specific to ha postgresql, or something missed from the kong hybrid stuff